lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -131,8 +131,6 @@ def __init__(
         pseudo_bbox = self.transLimits.inverted().transform([(0, 0), (1, 1)])
         self._pseudo_w, self._pseudo_h = pseudo_bbox[1] - pseudo_bbox[0]
 
-        self.figure.add_axes(self)
```

Comment:
Is this going to break basically every usage of 3D axes?

Reply:
No, only those using the rather unidiomatic `Axes3D(fig)` instead of `fig.add_subplot(..., projection="3d")`.